File: Intro to ASP.NET MVC 4/[VB]-Intro to ASP.NET MVC 4/VB/MvcMovie/Controllers/HomeController.vb
Public Class HomeController
    Inherits System.Web.Mvc.Controller

    Function Index() As ActionResult
        ViewData("Message") = "Modify this template to jump-start your ASP.NET MVC application."

        Return View()
    End Function

    Function About() As ActionResult
        ViewData("Message") = "Your quintessential app description page."

        Return View()
    End Function

    Function Contact() As ActionResult
        ViewData("Message") = "Your quintessential contact page."

        Return View()
    End Function
End Class

File: Locate Blood Donors using Online Maps/[C#]-Locate Blood Donors using Online Maps/C#/BloodDonors/Controllers/HomeController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;

namespace BloodDonors.Controllers
{
    public class HomeController : Controller
    {
        public ActionResult Index()
        {
            ViewBag.Message = "Find Blood Donors in your Area.";

            return View();
        }

        public ActionResult About()
        {
            ViewBag.Message = "Your quintessential app description page.";

            return View();
        }

        public ActionResult Contact()
        {
            ViewBag.Message = "Your quintessential contact page.";

            return View();
        }
    }
}
